lib/logflare/backends/adaptor.ex

```diff
@@ -23,7 +23,7 @@ defmodule Logflare.Backends.Adaptor do
       end
 
     %{
-      id: {adaptor_module, backend.id},
+      id: {adaptor_module, source.id, backend.id},
```

Comment:
i think for now we won't, as it would require too much refactoring of the BQ pipeline and extend timeline too much.
It would be easier to do so once we completely remove all the overhead that v1 brings.
Main goal of this interop PR is to allow us to do manual load testing and benchmarks so that we can get the performance on par or better than v1.